src/caret_analyze/plot/bokeh/pub_sub_info_interface.py

```diff
@@ -230,9 +235,11 @@ def _get_ts_column_name(
         pub_sub: Union[Publisher, Subscription]
     ) -> str:
         if isinstance(pub_sub, Publisher):
-            # TODO: fix here, since callback_names may take None as well.
-            ts_column_name = (f'{pub_sub.callback_names[0]}'
-                              '/rclcpp_publish_timestamp [ns]')
+            if pub_sub.callback_names:
+                ts_column_name = (f'{pub_sub.callback_names[0]}'
```

Comment:
`pub_sub.callback_names` may take `[]` which raises `IndexError: list index out of range.`

Reply:
```
File ~/ros2_caret_ws/build/caret_analyze/caret_analyze/plot/bokeh/pub_sub_info_interface.py:240, in PubSubTimeSeriesPlot._get_ts_column_name(self, pub_sub)
    234 def _get_ts_column_name(
    235     self,
    236     pub_sub: Union[Publisher, Subscription]
    237 ) -> str:
    238     if isinstance(pub_sub, Publisher):
    239         # TODO: fix here, since callback_names may take None as well.
--> 240         ts_column_name = (f'{pub_sub.callback_names[0]}'
    241                           '/rclcpp_publish_timestamp [ns]')
    242     else:
    243         ts_column_name = f'{pub_sub.column_names[0]} [ns]'

IndexError: list index out of range
```

Reply:
@atsushi421 
Could you confirm this error?

Reply:
@hsgwa 
In the case of `[]`, the current implementation can handle the situation, so no change is necessary.
Added test code to verify: https://github.com/tier4/CARET_analyze/pull/133/commits/62cc178d8083b6f8bfaaad8705838a053cfcf959